docs/etcd_clusters.md

```diff
@@ -0,0 +1,60 @@
+# Manage etcd clusters
+
+Mayu contains an etcd discovery to setup and manage your etcd clusters. During startup you can configure if you like to use an external etcd discovery or Mayu itself.
```

Comment:
I've changed the parameter behaviour. There is now a `--use-internal-etcd-discovery` flag and there is more validation around the parameters.